docs/guides/distribution/updater.md

```diff
@@ -224,113 +132,103 @@ The alternate update technique uses a plain JSON file, storing your update metad
     }
   }
 }
 ```
 
-Note that each platform key is in the `OS-ARCH` format, where `OS` is one of `linux`, `darwin` or `windows`, and `ARCH` is one of `x86_64`, `aarch64`, `i686` or `armv7`.
+The required keys are `"version"`, `"platforms.[target].url"` and `"platforms.[target].signature"`; the others are optional.
 
-## Bundler (Artifacts)
+- `"version"` must be a valid semver, with or without a leading `v`, meaning that both `1.0.0` and `v1.0.0` are valid.
+- `"platforms"`: Each platform key is in the `OS-ARCH` format, where `OS` is one of `linux`, `darwin` or `windows`, and `ARCH` is one of `x86_64`, `aarch64`, `i686` or `armv7`.
+- `"url"` must be a valid url to the update bundle.
+- `"signature"` must be the **content** of the generated `.sig` file. The signature may change each time you run `tauri build` so make sure to always update it.
+- `"notes"`: Here you can add notes about the update, like release notes. Tauri's default dialog will present this to the user when it asks if it's allowed to update.
+- `"pub_date"` must be formatted according to [RFC 3339] if present.
 
-The Tauri bundler automatically generates update artifacts if the updater is enabled in `tauri.conf.json`
-Your update artifacts are automatically signed if the bundler can locate your private and public keys.
+Note that Tauri will validate the _whole_ file before checking the version field, so make sure all existing platform configurations are valid and complete.
 
-The signature is the content of the generated `.sig` file. The signature can be uploaded to GitHub safely or made public if your private key is secure.
+## Checking for Updates
 
-You can see how it's [bundled with the CI][artifacts updater workflow] and a [sample tauri.conf.json].
-
-### macOS
-
-On macOS, we create a .tar.gz from the whole application. (.app)
+### Built-in dialog
 
-```
-target/release/bundle
-└── macos
-    └── app.app
-    └── app.app.tar.gz (update bundle)
-    └── app.app.tar.gz.sig
-```
+By default, the updater uses a built-in dialog API from Tauri. The dialog will only check for a new update when the app was just launched or when you manually [emit](../../api/js/event.md#emit) the `"tauri://update"` event.
 
-### Windows
+![New Update](https://i.imgur.com/UMilB5A.png)
 
-On Windows, we create a .zip from the MSI; when downloaded and validated, we run the MSI install.
+The dialog release notes are represented by the update `notes` provided by the [server](#server-support).
+If the user accepts, the update is downloaded and installed. Afterwards, the user is prompted to restart the application.
 
-```
-target/release/bundle
-└── msi
-    └── app.x64.msi
-    └── app.x64.msi.zip (update bundle)
-    └── app.x64.msi.zip.sig
-```
+### Custom Dialog and Logic
 
-### Linux
+:::caution
 
-On Linux, we create a .tar.gz from the AppImage.
+You need to disable the built-in dialog in your [Tauri configuration](#tauri-configuration) to enable the JavaScript APIs and updater events!
```

Comment:
Is there a more specific place to point to here?

Reply:
Either the tauri config section in the guide, or the `dialog` property in config.md. In this case i'd be fine with both